Cargo.toml

```diff
@@ -17,3 +17,6 @@ members = [
   "tower-service-util",
   "tower-timeout",
 ]
+
+[patch.'https://github.com/tower-rs/tower']
+tower-retry = { path = "tower-retry" }
```

Comment:
I think you mentioned this in gitter, but why is this patch statement needed? In theory, patch statements only apply to the git repo proper and won't apply to crates depending on tower via git?

Reply:
We currently use tower-hyper's fixed retry policy that depends on the git version but locally here we are using the path version and thus there is a trait type mismatch, which this patch solves. We just need to release `tower-retry` on crates and that should solve this patch.